_posts/2020-06-24-Join-the-BIDS-community-on-the-BrainHack-Mattermost.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Join the BIDS community on the BrainHack Mattermost
+author: Franklin Feingold
+display: true
+---
+
+# Join the BIDS community on the BrainHack Mattermost
+
+BIDS is joining the [BrainHack Mattermost](https://mattermost.brainhack.org/){:target:"_blank"}!
+
+<!--more-->
+
+We are proud to announce our home for hosting BIDS organization instant messaging: the [BrainHack Mattermost platform](https://mattermost.brainhack.org/){:target:"_blank"}! In the past few years, several BIDS channels have independently been created on Mattermost by the community. The Steering Group and Maintainers Group have choosen to recognize the BrainHack Mattermost as a member of our organizations's official communication suite. This platform will live alongside our other official communication channels: NeuroStars (forum for support questions), mailing list (announcements), Twitter (community news), and GitHub repositories (bug reports + feature requests). 
```

Comment:
- fix typo choosen -> chosen
- each sentence on new line for better diff
- turn communication chans into list

```suggestion
We are proud to announce our home for hosting BIDS organization instant messaging: the [BrainHack Mattermost platform](https://mattermost.brainhack.org/){:target:"_blank"}! In the past few years, several BIDS channels have independently been created on Mattermost by the community.
The Steering Group and Maintainers Group have chosen to recognize the BrainHack Mattermost as a member of our organizations's official communication suite.
This platform will live alongside our other official communication channels:

-  NeuroStars (forum for support questions)
-  mailing list (announcements)
-  Twitter (community news)
-  and GitHub repositories (bug reports + feature requests). 
```